contracts/Compass.vy

```diff
@@ -76,6 +77,12 @@ event ERC20DeployedEvent:
     decimals: uint8
     event_id: uint256
 
+event ArbitraryEvent:
```

Comment:
Can we name this something more specific to node sale? For instance `NodeSaleEvent`?

Reply:
@webelf101 please respond.